app/src/main/java/org/eyeseetea/malariacare/strategies/ALoginActivityStrategy.java

```diff
@@ -53,6 +56,26 @@ public void checkCredentials(Credentials credentials, Callback callback) {
         callback.onSuccessDoLogin();
     }
 
+    public EditText initProgramServer(){
```

Comment:
It's better does not return null and in caller is not used return value, here we can refactor this method to void